src/pages/features.js

```diff
@@ -72,11 +72,11 @@ const FeaturesPage = () => (
             </article>
 
             <article>
-              <h2>Pivotal Data Products</h2>
+              <h2>VMware Data Products</h2>
               <p>
-                Ready to connect with Pivotal Data Products including
-                Gemfire/Geode, PCC, GPDB, Pivotal RabbitMQ, Pivotal Redis,
-                Pivotal PostgreSQL, and Pivotal MySQL
+                Ready to connect with VMware Data Products including
```

Comment:
Many of the products still include Pivotal branding; see: https://www.vmware.com/products/pivotal-rabbitmq.html